scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -52,16 +45,15 @@ import scala.collection.convert.Wrappers
 import scala.collection.mutable
 
 private object KryoRegistrarLoader {
-
-  private[this] val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
It seems like in scio's scala files we mostly use `logger`, let's keep it `logger` and if you wanna rename pls submit a PR with holistic change.